bodhi/server/consumers/masher.py

```diff
@@ -979,3 +980,7 @@ def run(self):
         else:
             self.success = True
         return out, err, returncode
+
+
+def clean_up():
+    pass
```

Comment:
I dont' think we need a new function here. We just need to call ```clean_old_mashes()``` when the mash is done and successful.